src/pages/notifications/pause-feedback.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { mount, html } from 'hybrids';
+import '/ui/index.js';
+
+import { setupNotificationPage } from '/utils/notifications.js';
+import { REVIEW_PAGE_URL } from '/utils/urls.js';
+import { openTabWithUrl } from '/utils/tabs.js';
+
+const close = setupNotificationPage(360);
```

Comment:
```suggestion
const close = () => setupNotificationPage(360);
```
so it can be called `<ui-button type="success" onclick="${close}">`

Reply:
No. The `setupNotificationPage()` returns a function, you cannot wrap `close` as a function, you need a result of that function call. As the close takes an argument, you can't use it directly, so it's wrapped in an arrow function.